src/script/descriptor.cpp

```diff
@@ -914,6 +914,7 @@ std::unique_ptr<DescriptorImpl> ParseScript(uint32_t key_exp_index, Span<const c
             }
         }
         if (ctx == ParseScriptContext::P2SH) {
+            // This limits the maximum number of pubkeys to 15.
```

Comment:
Perhaps clarify that this depends on whether they're compressed or uncompressed (but it's 15 for all compressed)?